pkg/k8smanifest/verify.go

```diff
@@ -47,18 +47,25 @@ type SignatureVerifier interface {
 }
 
 func NewSignatureVerifier(objYAMLBytes []byte, imageRef string, pubkeyPath *string) SignatureVerifier {
+	i := &ImageSignatureVerifier{onMemoryCacheEnabled: true}
 	var annotations map[string]string
 	if imageRef == "" {
 		annotations = k8smnfutil.GetAnnotationsInYAML(objYAMLBytes)
 		if annoImageRef, ok := annotations[ImageRefAnnotationKey]; ok {
 			imageRef = annoImageRef
+			i.imageRef = imageRef
```

Comment:
could you check the location of line 56 to right after line 58 like this, please?
imageRef could come from method argument.
```
	if imageRef == "" {
		annotations = k8smnfutil.GetAnnotationsInYAML(objYAMLBytes)
		if annoImageRef, ok := annotations[ImageRefAnnotationKey]; ok {
			imageRef = annoImageRef
		}
	}
	i.imageRef = imageRef
```

Reply:
nice one, thank you so much @hirokuni-kitahara, I updated the code according to your review.

Reply:
thank you very much! I have approved the changes.

Reply:
thank you so much! @developer-guy @hirokuni-kitahara

Reply:
thank you so much, folks @yuji-watanabe-jp @hirokuni-kitahara 🥳🎉